src/test/java/com/xceptance/neodymium/tests/DriverArgumentsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.xceptance.neodymium.tests;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+
+@RunWith(NeodymiumRunner.class)
+@Browser("Chrome_headless")
+@Browser("FF_headless")
+public class DriverArgumentsTest extends NeodymiumTest
+{
+    private static String randomLogFileName = "target/" + UUID.randomUUID().toString() + ".log";
+
+    @BeforeClass
+    public static void createSettings()
+    {
+        Map<String, String> properties1 = new HashMap<>();
+        properties1.put("neodymium.webDriver.chrome.driverArguments", "--silent ; --log-path=" + randomLogFileName);
```

Comment:
The Issue has a list with possible arguments, but the issue is quite old, please check if everything is still valid for current browsers.